daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/unparsers/UState.scala

```diff
@@ -91,17 +89,15 @@ abstract class UState(
   dataProcArg: Maybe[DataProcessor],
   tunable: DaffodilTunables)
   extends ParseOrUnparseState(vbox, diagnosticsArg, dataProcArg, tunable)
-  with Cursor[InfosetAccessor] with ThrowsSDE with SavesErrorsAndWarnings
-  with FormatInfo {
+  with Cursor[InfosetAccessor] with ThrowsSDE with SavesErrorsAndWarnings {
+
+  def unparse1(unparser: Unparser): Unit
```

Comment:
Remove unparse1 method from UState. Fix in unparser so call back to ustate isn't needed.